src/ecs/commands/runCommandInContainer.ts

```diff
@@ -28,7 +29,9 @@ export async function runCommandInContainer(
 ): Promise<void> {
     getLogger().debug('RunCommandInContainer called for: %O', node.containerName)
     let result: 'Succeeded' | 'Failed' | 'Cancelled' = 'Cancelled'
-    const viewOutput = localize('AWS.command.ecs.runCommandInContainer.viewOutput', 'View Output')
+    const status = vscode.window.setStatusBarMessage(
```

Comment:
I'd move the assignment to right before `executeCommand`.
You'll need to use `let status: vscode.Disposable` instead but that's ok since we're not dependent on `status` existing.